include/openspace/events/event.h

```diff
@@ -397,20 +399,13 @@ struct EventLayerAdded : public Event {
     /**
      * Creates an instance of an EventLayerAdded event.
      *
-     * \param node_ The identifier of the globe to which the layer is added
-     * \param layerGroup_ The identifier of the layer group to which the layer is added
-     * \param layer_ The identifier of the layer that was added
+     * \param layer_ A pointer to the layer that was added
```

Comment:
The parameter seems to be called `uri_`, not `layer_`. And it's actually not a pointer

Reply:
Same for line 404